scripts/README.md

```diff
@@ -3,3 +3,4 @@
 Common script utilities which can help automate common tasks within a BigQuery data warehouse.
 
 * [billing](/scripts/billing) - Example queries over the GCP billing export to help get you started using the standard data export format
+* [jobs](/scripts/jobs) - Example queries over the INFORMATION_SCHEMA views for jobs
```

Comment:
Create an additional README in the jobs directory similar to billing.

Additionally given that this directory will likely eventually contain multiple SQL scripts. It may be good to create an additional SQL subdirectory.

Reply:
done